packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -143,7 +150,7 @@ export async function deployArmTemplates(ctx: SolutionContext): Promise<Result<v
   );
   const armTemplateJsonFilePath = path.join(azureInfraDir, templateFolder, armTemplateJsonFileName);
   await compileBicepToJson(orchestrationFilePath, armTemplateJsonFilePath);
-  ctx.logProvider?.info("[Solution] Successfully compile bicep files to JSON.");
+  ctx.logProvider?.info("Successfully compile bicep files to JSON.");
```

Comment:
Why we removed the prefix?

Reply:
Only the start and the end message will prefix "solution". Or do we have a consistent practice? @1yefuwang1  @jayzhang

Reply:
Will add the prefix.